manifest.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: apps/v1
```

Comment:
I think I would prefer not to include this in the repo? Seems a little specific - and this is just what we're doing with the charm. So long as the environment variables etc. are documented, I think you can omit Kubernetes specific config from this repo.

Reply:
Yes Jon. I removed it now.